deployment/deployutils/deployutils.cpp

```diff
@@ -332,6 +332,14 @@ void addItem(StringBuffer& jsStrBuf,
              const char* extra, 
              short ctrlType)
 {
+  // Control types:
```

Comment:
Thanks for adding this comment!
It might be outside of the scope of this issue, but it'd be nice to have an enumeration to represent these types.